pkg/extract/tar.go

```diff
@@ -58,12 +55,8 @@ func ExtractTarFile(buffer *bytes.Buffer, destDir string, fileSize int64) error
 		}
 	}
 	elapsed := time.Since(startTime).Seconds()
-	size := humanize.Bytes(uint64(fileSize))
-	throughput := humanize.Bytes(uint64(float64(fileSize) / elapsed))
 	logger.Info().
-		Str("size", size).
 		Str("elapsed", fmt.Sprintf("%.3fs", elapsed)).
-		Str("throughput", throughput).
 		Msg("Extracted")
```

Comment:
For now lets just drop this log. We can refactor this whole thing to be better (down the road) and add extra debugging such as "creating directory", "extracting file", "creating symlink", etc etc.

That is assuming we make the change below making the DownloadFile responsible for the final .Info() logging.